app.py

```diff
@@ -1,40 +1,54 @@
 from flask import Flask, render_template, request, redirect, url_for
-import trello_items as trello
+import db_items as mongoDB
 from viewmodel import ViewModel
 from app_config import Config
+import pymongo
+import certifi
+import os
 
 def create_app():
     app = Flask(__name__) 
     app.config.from_object(Config())
+
+    board_id = os.getenv('BOARD_ID')
+    db_username = os.getenv('MONGO_USERNAME')
+    db_password = os.getenv('MONGO_PASSWORD')
+
+    client = pymongo.MongoClient(
+        "mongodb+srv://{}:{}@cluster0.omkjb.mongodb.net/todoDB?retryWrites=true&w=majority".format(db_username, db_password), 
+        tlsCAFile=certifi.where()
+    )
+    db = client.TodoListDB
+    collection = db.todos
     
     @app.route('/')
     def index():
-        items = trello.get_items()
+        items = mongoDB.get_items(collection, board_id)
         item_view_model = ViewModel(items[0], items[1], items[2])
         return render_template('index.html', view_model=item_view_model)
 
     @app.route('/add', methods=['POST'])
     def add():
         name = request.form.get('new_item_name')
         description = request.form.get('new_item_description')
-        trello.create_item(name, description)
+        mongoDB.create_item(collection, board_id, name, description)
         return redirect(url_for('index'))
 
     @app.route('/start/<item_id>', methods=['POST'])
     def start_item(item_id):
-        trello.start_item(item_id)
+        mongoDB.start_item(collection, board_id, item_id)
         return redirect(url_for('index'))
 
     @app.route('/complete/<item_id>', methods=['POST'])
     def complete_item(item_id):
-        trello.complete_item(item_id)
+        mongoDB.complete_item(collection, board_id, item_id)
         return redirect(url_for('index'))
 
     @app.route('/undo/<item_id>', methods=['POST'])
     def undo_item(item_id):
-        trello.undo_item(item_id)
+        mongoDB.undo_item(collection, board_id, item_id)
         return redirect(url_for('index'))
 
-    return app
+    return app, collection
```

Comment:
This pattern of returning two things doesn't match gunicorn's expectations, so causes your production docker container to error. A simple solution would be to ask gunicorn to run the app variable that this function returns below, rather than calling create_app explicitly e.g. updating your entrypoint file to `poetry run gunicorn "app:app" --bind 0.0.0.0:$PORT`

Reply:
Done